specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,126 @@
+# Service Subcategory
+
+The Service Subcategory is the second-highest-level classification of a [*service*](#glossary:service) based on its core function.  The Service Subcategory (in conjunction with the Service Category) is commonly used for scenarios like analyzing costs across providers and tracking the migration of workloads across fundamentally different architectures.  The Service Subcategory is a child of the Service Category, and each Service Subcategory value must have one and only one Service Category parent. Though a given *service* can have multiple purposes, each *service* should have one and only one Subcategory that best aligns with its primary purpose. 
+
+The ServiceSubcategory column MUST be present in a FOCUS dataset and MUST NOT be null. This column is of type String and MUST be one of the allowed values.
+
+## Column ID
+
+ServiceSubcategory
+
+## Display Name
+
+Service Subcategory
+
+## Description
+
+Second-highest-level classification of a *service* based on its core function.
+
+## Content Constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column type     | Dimension      |
+| Feature level   | Mandatory      |
```

Comment:
We discussed whether to make this column mandatory or optional in today's Member call; results were inconclusive.  Please weigh in with your thoughts!

Reply:
The Service Category is Mandatory, so to be consistent and simple, I would go for the first option:

> Mandatory and allow providers to leave it empty OR use 'Other ($ServiceCategory)'

Reply:
Mandatory

Reply:
I'm getting a pretty strong sense of Mandatory from folks here (and that's what I think too).  Will leave this open until early next week and try to get more opinions in the meantime!

Reply:
Just on the description do we need the word 'granular' . To me the Service Category or Service Sub-Category are hierarchial elements and I would say that it provides further classifications but I am not sure of the word 'granular' . Also on non-normative descriptions do we leave must as lowercase or should be in caps?

Reply:
I shouldn't have said "empty or use 'Other ($Service Category)' ". Here I think null and empty are the same and since we're saying null is not allowed, basically providers would have to only use 'Other ($Service Category)' if they don't want to support this right away?

Reply:
Discussed this point on today's TF1 call. Results remain inconclusive, wanting to balance provider willingness/ability to populate the column vs the practitioner's need to have a fully populated dataset.  

I will also point out that the Service Category column is mandatory, so I feel it would be good to be consistent across these two columns.

Udam's request was for me to check with Tim on this point, which I shall do.

Reply:
I vote for "Mandatory" with "Other" as the default

Reply:
Mandatory non-null is best for practitioners.

For the sanctity of the column, optional non-null would give the most consistent results. (When there's a value, you know it's good.)

If we allow mandatory nullable, practitioners won't know when to expect a value and when not. There's no contract with the provider to get a complete dataset.

If we allow mandatory with "Other" as a fallback, practitioners won't be able to tell if something should be in Other or not. This will create a huge data cleanup effort for all the historical data that doesn't have this until the provider in question implements it. That seems like the worst option 😕

Reply:
We should have this column be either Optional (ie provider may chose to include the column(s) or not), or  Mandatory nullable. Otherwise we create the expectation with practitioners that the column will be be present and/or populated, and when it isn't the FOCUS product will appear to be broken. Providers can communicate whether the column will be present or not, populated or not in their focus data sets in public documentation.

Reply:
I vote for Mandatory nullable simply because it being in the dataset will force practitioners to know they should expect values here and take their feedback to the providers. Optional doesn't do justice for how much value this categorization provides - especially when analyzing cost across multiple clouds.

Reply:
Mandatory nullable appears to have the strongest acceptance from the group, both here in the comments and in the last couple of FOCUS meetings.  I will change it to that, and we can debate further as necessary.

Reply:
Discussed in today's Maintainer meeting; grouped decided to make the column Optional, but there is not yet alignment on nullability.  I issued a poll in the FOCUS Slack, and I will resolve this thread with the outcome in the next 24 hours.

Reply:
After MUCH deliberation, we have aligned on moving forward with Optional.  Committing the below suggestion and resolving.

```suggestion
| Feature level   | Optional      |
```